File: examples/eApi v1.0/java/eAPI-v10-hu-connector/src/main/java/cz/monetplus/mips/eapi/v10hu/connector/entity/PaymentReverseRequest.java
package cz.monetplus.mips.eapi.v10hu.connector.entity;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NonNull;
import lombok.RequiredArgsConstructor;

@EqualsAndHashCode(callSuper = false)
@Data
@RequiredArgsConstructor
public class PaymentReverseRequest  extends SignBase  {
    @NonNull
    private String merchantId;
    @NonNull
    private String payId;

    @Override
    public String toSign() {
        StringBuilder sb = new StringBuilder();
        add(sb, merchantId);
        add(sb, payId);
        add(sb, dttm);
        deleteLast(sb);
        return sb.toString();
    }
}

File: examples/eApi v1.0/dotNET/KHGTWDemo/KHGatewayClientExample/Communication/DataObjects/PaymentReverseRequest.cs
using System.Text;

namespace KHGatewayClientExample.Communication.DataObjects;

public class PaymentReverseRequest : PayReq
{
    public override string ToSign()
    {
        var sb = new StringBuilder();
        Add(sb, MerchantId);
        Add(sb, PayId);
        Add(sb, Dttm);
        DeleteLast(sb);
        return sb.ToString();
    }
}